hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/dynamicconfig/MapStoreConfigHolder.java

```diff
@@ -181,6 +198,7 @@ public static MapStoreConfigHolder of(MapStoreConfig config, SerializationServic
         holder.setWriteBatchSize(config.getWriteBatchSize());
         holder.setWriteCoalescing(config.isWriteCoalescing());
         holder.setWriteDelaySeconds(config.getWriteDelaySeconds());
+        holder.setOffload(config.isOffload());
```

Comment:
shouldn't holder. isOffloadExists be set here as well?

Reply:
no, it is static utility to copy existing map-store-config, there is no state for `isOffloadExists`